src/mesh_analysis/readers/victoria_reader.py

```diff
@@ -74,3 +74,20 @@ def single_query_info(self) -> Result[Dict, Response]:
             logger.error(f'Failed to decode JSON: {e}')
             logger.error(f'Response content: {response.content}')
 
+            return Err(response)
+
+    def multi_query_info(self) -> Result[Iterator, str]:
+        time.sleep(10)
```

Comment:
There was an "unknown" problem, sometimes requesting data from the lab triggered a max request error. This could be fixed increasing the limit from the server side, but it is not done yet.